src/Serialize.h

```diff
@@ -239,7 +239,7 @@ namespace drafter {
         }
 
         template<typename Collection, typename Functor, typename Arg1, typename Arg2>
-        R operator()(const Collection& collection, Functor &wrapper, Arg1 arg1, Arg2 arg2) const {
+        R operator()(const Collection& collection, Functor &wrapper, Arg1 arg1, Arg2& arg2) const {
```

Comment:
Because this arg is used for `context` and if we don't pass it by reference then the warnings we add will just disappear.